src/glossary.rst

```diff
@@ -1694,11 +1740,11 @@ Glossary
    runahead limit
    runahead
       In a :term:`cycling workflow`, the runahead limit determines how far
-      ahead of the oldest :ref:`active cycle point` the workflow is permitted
+      ahead of the oldest :term:`active cycle point` the workflow is permitted
```

Comment:
Personal preference (cycles are a clearer concept without the "point" bit):

```suggestion
      ahead of the oldest :term:`active cycle` the workflow is permitted
```

Reply:
I prefer "cycle point" for clarity - it distinguishes between the sequence as a whole (e.g., a "yearly cycle") and the point value ("2034"). 

Also, I think we still have a few user-facing (albeit possibly deprecated) options and arguments called "point" rather than "cycle".

I'll concede that sometimes, as in this example, "point" can be probably be omitted because the context makes it clear.